packages/form-validation/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@auroformkit/form-validation",
+  "description": "Shared validation for all form components in auro-formkit monorepo",
+  "version": "0.0.0",
+  "private": true,
+  "type": "module",
+  "main": "./src/validation.js",
+  "types": "tsc",
+  "exports": {
+    ".": "./src/validation.js"
+  }
```

Comment:
For `exports`, we may want to do this:

```
{
  "exports": {
    ".": {
      "types": "./dist/index.d.ts",
      "default": "./dist/index.js"
    },
    "./formValidation": {
      "types": "./dist/formValidation.d.ts",
      "default": "./dist/formValidation.js"
    }
  }
}
```

Then create `src/index.js` with:

```
export { default as formValidation } from './formValidation.js';
```

That would maintain the convention in other packages and allow for:

```
import { formValidation } from '@auro-formkit/form-validation';
```

(But be sure to test, I haven't tried it locally)